flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlDdlToOperationConverterTest.java

```diff
@@ -2603,6 +2626,72 @@ private CatalogTable prepareTable(
         return catalogTable;
     }
 
+    private CatalogMaterializedTable prepareMaterializedTable(
```

Comment:
this method was also introduced in https://github.com/apache/flink/pull/26990

so it will disappear from this PR after https://github.com/apache/flink/pull/26990 is merged and this one will be rebased